Core/src/OffConverter/OffModel.cpp

```diff
@@ -26,13 +26,15 @@ void OpenInfraPlatform::Core::OffConverter::OffModel::reset()
 
 void OpenInfraPlatform::Core::OffConverter::OffModel::addVertices(const std::vector<buw::Vector3d>& vertices)
 {
+	geometry_.vertices.reserve(vertices.size());
 	for (const auto& vertex : vertices)
 		geometry_.vertices.push_back(vertex);
 	geometry_.UpdateBBox();
 }
 
 void OpenInfraPlatform::Core::OffConverter::OffModel::addIndices(const std::vector<uint32_t>& indices)
 {
+	geometry_.indices.reserve(indices.size());
```

Comment:
Is `addIndices` function called only once? Otherwise, the `.reserve` will overwrite the contents of the `.indices` every time it is called - I'M not sure this is a wanted effect.

Reply:
Yes, it is only called once at the very end of the reading process.